trendmicro_apex/plugin.spec.yaml

```diff
@@ -126,3 +718,546 @@ actions:
+        type: PermissionCtrl
+        required: false
+      SystemCtrl:
+        title: "System CTRL"
+        description: Indicates the suspicious object distribution role of the Apex Central server
+        type: SystemCtrl
+        required: false
+  get_rca_object:
+    title: Get RCA Response Object
+    description: Retrieves results of investigations in different formats based on the specified taskId
+    input:
+      task_id:
+        title: Task ID
+        description: Task ID from another API call that is used to retrieve a specific task result Specify the taskId values returned by the following APIs
+        type: string
+        required: true
+        example: "9BD2204C-0554-45C8-9C62-799284928AFA"
+      task_type:
+        title: Task Type
+        description: Type of API request. For Endpoint Sensor, the value is always 4
+        type: string
+        required: false
+        default: CMEF
+        example: "CMEF"
+        enum:
+          - UNKNOWN
+          - INTERNAL
+          - CM
+          - CMEF
+          - OSF_COMMAND
+          - OSF_QUERY
+          - OSF_NOTIFY
+          - OSF_LOG
+          - MDR_ATTACK_DISCOVERY
+          - OSF_SYS_CALL
+      content_id:
+        title: Content ID
+        description: Indicates the location of the dataset
+        type: string
+        required: false
+        example: 8
+      limit:
+        title: Limit
+        description: Retrieves the top n results from the servers
+        type: string
+        required: true
+        example: "1"
+    output:
+      Data:
+        title: Data
+        description: Contains the data returned by the specific API
+        type: api_data
+        required: true
+      FeatureCtrl:
+        title: "Feature CTRL"
+        description: The Apex Central deployment model
+        type: FeatureCtrl
+        required: false
+      Meta:
+        title: "Meta"
+        description: Indicates the response status, including the result, error code, and error message
+        type: Meta
+        required: false
+      PermissionCtrl:
+        title: "Permission CTRL"
+        description: Indicates the permissions assigned to the logged-on user account for accessing Apex Central menu items and features
+        type: PermissionCtrl
+        required: false
+      SystemCtrl:
+        title: "System CTRL"
+        description: Indicates the suspicious object distribution role of the Apex Central server
+        type: SystemCtrl
+        required: false
+  terminate_process:
+    title: Terminate Process
+    description: Terminates the processes specified on Security Agent endpoints
+    input:
+      agent_guid:
+        title: Agent GUID
+        description: The key is the serverGuid, and the value is a list of agentGuid strings of the endpoints managed by the target server
+        type: object
+        required: false
+        example: '{"2EBEC86D-3FEB-4666-9CA6-B80AB1E193E6": ["B7F34FF9-9DEE-45F4-9F24-DC5116C79D52"]}'
```

Comment:
This is required by API. Agents are connected to specific server_guid. So it should be mapped:
server_guid -> agent_guid.
In API there say: "If both agentGuid and serverGuid contain an empty array, the termination request targets all agents".

So maybe we can set server_guid always to empty array. In my opinion in this way we will send request to all servers and then every one of them will filter agents by guid.

But in my opinion there is better to get server_guids from agent_guids dict and set server_guids field in API request. We will add it in this way.

Reply:
@mgorny-morecode For user-friendliness, the solution should avoid a user from having to configure a JSON object as input to make it simple.

It looks like the user already specifies `server_guid` on line 1115. Can we complete the object for them in the code? e.g.
obj[server_guid] = agent_guid

We will want to not allow this to happen:  "the termination request targets all agents." because it's a dangerous operation. I agree, we should require a service_guid and list of agents but I think we can make it easier by only setting the server_guid one time in the whole configuration.